### ISSUE TITLE
BetterWardrobe Appearances Tab is completely blank

### ISSUE THREAD

**Issue by NZephyr** [SLOKnightfall/BetterWardrobe#287]
[BetterWardrobeError-1-BW_DBSavedSetDropdown.txt](https://github.com/SLOKnightfall/BetterWardrobe/files/9885236/BetterWardrobeError-1-BW_DBSavedSetDropdown.txt)
[BetterWardrobeError-2-BetterWardrobe-CanIMogItPlugin.txt](https://github.com/SLOKnightfall/BetterWardrobe/files/9885237/BetterWardrobeError-2-BetterWardrobe-CanIMogItPlugin.txt)
![Capture_BlizzUI](https://user-images.githubusercontent.com/65876278/198501894-5be47dba-7ee7-41e3-a559-473b3a047a45.PNG)
![Capture_ElvUI](https://user-images.githubusercontent.com/65876278/198501895-407a13d1-68e1-47c0-a24e-f4dd87478de0.PNG)

Files attached of the two errors that BugGrabber/BugSack caught and a Screen of both the ElvUI appearance issue, and the default BlizzUI to show it's not just a conflict with ElvUI (my brain is too tired to dig into the error code to see if ElvUI conflicts are in it, sorry!)

Basically, the Appearances tab of the Collections window is completely blank when this mod is active. I checked, and my BW is up to date (4.0.1). I think the DBSavedSetDropdown error is the relevant one, but I included the CanIMogIt plugin error in case it's related somehow? (I'm not sure how it would be, but weirder things have happened!)

The Dressing Room seems to work (at least the options I use). The Transmog vendor menu (where you actually apply the transmog, I can't recall if that's the proper term) SOMETIMES loads, but only the first tab (Items) and none of the other tabs change anything. Other times, it's blank completely, save for the dressing room of the character/item slots on the far left.

**Comment by eivl** [comment]
thats the issue i had. check my issue, maybe they are related. 

https://github.com/SLOKnightfall/BetterWardrobe/issues/289

**Comment by SLOKnightfall** [comment]
Fixed in the 4.1 update
